sdks/go/README.md

```diff
@@ -156,8 +156,9 @@ are added in a PR then the lock file needs to be updated.
 From the `$GOPATH/src/github.com/apache/beam` directory run
 
 ```
+$ ./gradlew :sdks:go:goVendor
 $ ./gradlew :sdks:go:goLock
-`./gradlew :goPostcommit`
+$ ./gradlew :goPostcommit
```

Comment:
We should probably change this to run the portable precommit test suite instead of the Dataflow using one.

Reply:
If the portable precommit command doesn't work that easily, file a jira for the issue and then leave this change as is. No sense scope creeping this PR.

Reply:
Done.